src/proposal.mligo

```diff
@@ -183,7 +183,7 @@ let add_proposal (propose_params, store : propose_params * storage): storage =
     ; metadata = propose_params.proposal_metadata
     ; proposer = Tezos.sender
     ; proposer_frozen_token = propose_params.frozen_token
-    ; proposer_fixed_fee_in_token = store.fixed_proposal_fee_in_token
+    ; proposer_fixed_fee_in_token = config.fixed_proposal_fee_in_token
```

Comment:
This `proposer_fixed_fee_in_token` can now be removed and this operation is no longer necessary.

The reason `proposer_fixed_fee_in_token` exists is that before a proposal was flushed it was possible for the admin to change the fixed fee so it needed to be stored.

Since that's no longer a possibility we can just use the one from the `config` where needed.

Reply:
Removed